tutorials/plot_sims.py

```diff
@@ -8,43 +8,120 @@
 """
 
 ###################################################################################################
+#
 # Simulation spiking data
 # -----------------------
 #
-# Words, words, words.
+# Sections
+# ~~~~~~~~
+#
+# This tutorial contains the following sections:
+#
+# 1. Simulate spikes based on spiking probabilities
+# 2. Simulate spikes based on different probability distributions
+# 3. Utilities for working with simulated spiking data
 #
 
 ###################################################################################################
 
 # sphinx_gallery_thumbnail_number = 1
 
-import spiketools
+# import auxiliary libraries
+import numpy as np
+import matplotlib.pyplot as plt
+import random
+
+# import all functions from spiketools.sim and a supporting function from spiktool.measures
+from spiketools.sim.dist import sim_spiketrain_binom, sim_spiketrain_poisson
+from spiketools.sim.prob import sim_spiketrain_prob
+from spiketools.sim.utils import refractory
+from spiketools.measures.conversions import convert_train_to_times
+
+###################################################################################################
+#
+# 1. Simulate spikes based on spiking probabilities
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
 
 ###################################################################################################
 
-# Settings
-...
+# Simulate spike train of size n_samples based on a probability of spiking per sample
+p_spiking = 0.7
+n_samples = 30
+sim_spiketrain = sim_spiketrain_prob(p_spiking, n_samples)
 
 ###################################################################################################
-# Subtitle
-# ~~~~~~~~
+
+# Convert binary spike train to spike times in milliseconds & plot the spike times
+spike_times = convert_train_to_times(sim_spiketrain)
+plt.eventplot(spike_times)
+
+###################################################################################################
+
+# Additionally, we can simulate spike train of size n_samples, based on a probability of spiking per sample.
+p_spiking = np.array([random.random() for i in range(1000)])
+sim_spiketrain = sim_spiketrain_prob(p_spiking)
+
+###################################################################################################
+
+# Convert binary spike train to spike times in milliseconds
+spike_times = convert_train_to_times(sim_spiketrain)
+
+# Plot the probability distribution of spike times
```

Comment:
if you import `plot_hist` from `spiketools.plts.data`, you can use that here. 

One benefit of doing to is there is a helper function to manage things like titles and labels, so you can do something like the following:
```
plot_hist(spike_times, density=1, bins=50, 
               xlabel='Spike Times (ms)', ylabel='Probability', 
               title='Probability Distribution of Spike Times')
```